arc-script/arc-script-test/compile/src/snapshots/ast@structs.arc.snap

```diff
@@ -4,8 +4,12 @@ expression: s
 input_file: arc-script-test/compile/src/tests/expect_pass/structs.arc
 
 ---
+fun foo(a: { c: i32, b: i32 }, b: { b: i32, c: i32 }) -> { a: { c: i32, b: i32 }, xyz: i32, d: { b: i32, c: i32 } } {
```

Comment:
Why does a patch implementing enums change golden output for structs? Is this something that should go in a separate patch?

Reply:
strange, I split this into a separate commit now